docs/quickstart.html

```diff
@@ -154,9 +154,9 @@ <h4 class="anchor-heading">
             By default, each line you enter will result in a separate event being written to the topic.
         </p>
 
-        <pre class="line-numbers"><code class="language-bash">$ bin/kafka-console-producer.sh --topic quickstart-events --bootstrap-server localhost:9092
-This is my first event
-This is my second event</code></pre>
+        <pre class="language-bash"><code class="language-bash">$ bin/kafka-console-producer.sh --topic quickstart-events --bootstrap-server localhost:9092
+$ This is my first event
+$ This is my second event</code></pre>
```

Comment:
`kafka-console-producer.sh` has its own prompt using a `>` character and without a space.

```suggestion
>This is my first event
>This is my second event</code></pre>
```

Reply:
will update these

Reply:
resolved